server/testdb/data_stores.go

```diff
@@ -225,36 +223,12 @@ func (td *postgresDB) readDataStoreRow(ctx context.Context, row scanner) (model.
 			return model.DataStore{}, fmt.Errorf("cannot parse data store: %w", err)
 		}
 
-		return patchDataStoreValues(dataStore), nil
+		return dataStore, nil
 	default:
 		return model.DataStore{}, err
 	}
 }
 
-func patchDataStoreValues(dataStore model.DataStore) model.DataStore {
```

Comment:
I guess this is now covered by the `nil` checks from the models right?

Reply:
Forgot to delete this, thanks

Reply:
Yes, Otherwise it's just the same as using direct values instead of references